topics/go/algorithms/data/hash/doc.go

```diff
@@ -29,3 +27,4 @@ Hash table
   are hashed to the same bucket
 - See more at https://en.wikipedia.org/wiki/Hash_table
 */
+package hash
```

Comment:
golint is going to complain with the first line of documentation here doesn't say

Package hash something something something.

Reply:
Love This!!!

Reply:
Thanks & done!